tests/cases/fourslash/switchCaseCompletionPriority.ts

```diff
@@ -0,0 +1,126 @@
+/// <reference path="fourslash.ts" />
+
+// @noLib: true
+
+//// declare const thing: string;
+
+//// // Basic switch block - should prioritize case/default
+//// switch (thing) {
+////     /*basic*/
+//// }
+
+//// // Same - show all completions (not at switch body level)
+//// switch (thing) {
+////   case 42:
+////   /*sameAll1*/
+//// }
+
+//// // Same - show all completions (after break at same column as case)
+//// switch (thing) {
+////   case 42:
+////   break;
+////   /*sameAll2*/
+//// }
+
+//// // Same - show all completions (complex nested structure)
+//// switch (thing) {
+////   case 42:
+////   if (Math.random()) {
+////   }
+////   else {
+////   }
+////   /*sameAll3*/
+//// }
+
+//// // After break statement with proper indentation
+//// switch (thing) {
+////   case 42:
+////     break;
+////   /*newPrio1*/
+//// }
+
+//// // After same-line break statement
+//// switch (thing) {
+////   case 42: break;
+////   /*newPrio2*/
+//// }
+
+//// // After return statement with proper indentation
+//// switch (thing) {
+////   case 42:
+////     return 1;
+////   /*newPrio3*/
+//// }
+
+//// // After throw statement with proper indentation
+//// switch (thing) {
+////   case 42:
+////     throw new Error();
+////   /*newPrio4*/
+//// }
+
+// Scenarios where case/default should be prioritized
+goTo.marker("basic");
+verify.completions({
+    includes: [
+        { name: "case", sortText: "10" },
+        { name: "default", sortText: "10" }
+    ]
+});
+
+goTo.marker("newPrio1");
+verify.completions({
```

Comment:
Go back to baselining if the baselines aren't massive, and show some characteristics about the ordering. The key thing the tests need to witness is whether `case`, `default`, and `thing` appear in the list and what their relative order would be.